packages/fxa-settings/src/components/NotificationPromoBanner/index.tsx

```diff
@@ -34,7 +39,7 @@ const NotificationPromoBanner = ({
   // This value is used to animate the banner out of view when the user dismisses it.
   const [isClosing, setIsClosing] = useState(false);
 
-  const notificationBannerClosedLocalStorageKey = `__fxa_storage.fxa_disable_notification_banner.${dismissKey}`;
+  const notificationBannerClosedLocalStorageKey = PromoKeys[dismissKey];
```

Comment:
Noted below, but this key should be distinct from the key used for the "do it later" button in the inline flow.